### PR TITLE
adds verb to auto bunker override on linked servers

### PR DESCRIPTION
**Pull request by silicons**
## About The Pull Request

adds a verb that if configured allows players to 

## Why It's Good For The Game

having to fill out two access requests is silly

## Changelog
:cl:
add: auto bunker override verb has been added
/:cl:

this code is not tested

### CODE REVIEW

File: code/modules/client/verbs/autobunker.dm [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/client/verb/bunker_auto_authorize()
+	set name = "Auto Authorize Panic Bunker"
+	set desc = "Authorizes your account in the panic bunker of any servers connected to this function."
+	set category = "OOC"
+	
+	var/static/lastuse = 0
```
Comment (Ghommie) [inline code/modules/client/verbs/autobunker.dm]: This way, one guy could perpetually run the verb every 5 seconds and deny it to everyone else.

File: code/modules/client/verbs/autobunker.dm [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+	message["ckey"] = key
+	message["source"] = "[CONFIG_GET(string/cross_comms_name)]"
+	message["key"] = comms_key
+	message["auto_bunker_override"] = TRUE
+	var/list/servers = CONFIG_GET(keyed_list/cross_server_bunker_override)
+	if(!length(servers))
+		to_chat(C, "<span class='boldwarning'>AUTOBUNKER: No servers are configured to receive from this one.</span>")
+		return
+	var/logtext = "[key] ([key_name(C)]) has initiated an autobunker authentication with linked servers."
+	message_admins(logtext)
```
Comment (Ghommie) [inline code/modules/client/verbs/autobunker.dm]: ...And harrass admins about ittoo. Perhaps they should only be messaged on success.,,

**Review by Ghommie — CHANGES_REQUESTED** [review]

File: code/modules/client/verbs/autobunker.dm (hunk 1 above)
Comment (silicons) [reply code/modules/client/verbs/autobunker.dm]: @Ghommie it's logged + notify admin'd, i dont' want people able to send more than one topic every once in a while

File: code/modules/client/verbs/autobunker.dm (hunk 2 above)
Comment (silicons) [reply code/modules/client/verbs/autobunker.dm]: it's logged + message so admins know when someone is doing it 

admins can tell them to stop spamming 

hm

File: code/modules/client/verbs/autobunker.dm (hunk 1 above)
Comment (silicons) [reply code/modules/client/verbs/autobunker.dm]: well eh alternative is a client var and i'm not sure if we want that kind of bloat

File: code/modules/client/verbs/autobunker.dm (hunk 2 above)
Comment (Ghommie) [reply code/modules/client/verbs/autobunker.dm]: I'm unwilling to push the need of constant server and game flow monitoring from admins furthermore.

Comment (silicons) [reply code/modules/client/verbs/autobunker.dm]: yeah sure

**Comment by silicons** [comment]
@Ghommie i made it just log without messaging and client var for delay

**Review by Ghommie — APPROVED** [review]

**Pull request merged**